pkg/app/piped/cloudprovider/ecs/client.go

```diff
@@ -0,0 +1,182 @@
+// Copyright 2021 The PipeCD Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ecs
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/credentials/stscreds"
+	"github.com/aws/aws-sdk-go-v2/service/ecs"
+	"github.com/aws/aws-sdk-go-v2/service/ecs/types"
+	"go.uber.org/zap"
+)
+
+type client struct {
+	region string
+	client *ecs.Client
+	logger *zap.Logger
+}
+
+func newClient(region, profile, credentialsFile, roleARN, tokenPath string, logger *zap.Logger) (*client, error) {
+	if region == "" {
+		return nil, fmt.Errorf("region is required field")
+	}
+
+	c := &client{
+		region: region,
```

Comment:
nits, looks like this `c.region` is not be used anywhere else so I think it's safe to be removed 👀

Reply:
I've just removed it!